rct229/rules/section6/section6rule7.py

```diff
@@ -0,0 +1,94 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+
+MSG_WARN_DAYLIGHT_NO_SCHEDULE = "Some of the spaces in zone are modeled with window(s) and/or skylight(s) and have daylighting controls modeled explicitly in the simulation tool. Verify that the mandatory lighting control requirements are met."
+MSG_WARN_DAYLIGHT = "Some of the spaces in zone are modeled with window(s) and/or skylight(s) and have daylighting controls modeled via schedule adjustment. Verify that the mandatory lighting control requirements are met, and that the supporting documentation is provided for the schedule adjustment."
+MSG_WARN_NO_DAYLIGHT = "Some of the spaces in zone are modeled with fenestration but no daylighting controls. The design must include mandatory daylighting controls unless any of the exceptions to 90.1 section 9.4.1.1€ apply."
+
+DAYLIGHT_CONTROL_TYPE = schema_enums["LightingDaylightingControlType"].NONE
```

Comment:
It might more self documenting to set 
```
DAYLIGHT_CONTROL_TYPE = schema_enums["LightingDaylightingControlType"]
```
here and then use DAYLIGHT_CONTROL_TYPE.NONE where this is used in the code.

Reply:
Great point! This part was removed because we now use ```find_all``` function instead of for-loop.